tests/unit/query_graph/test_graph_quick_pruning.py

```diff
@@ -0,0 +1,77 @@
+"""
+Test for graph quick pruning related logics
+"""
+import pytest
+
+from featurebyte.query_graph.graph import QueryGraph
+
+
+def check_quick_pruned_graph(original_graph, quick_pruned_graph, target_nodes, mapped_node_names):
+    """Check the quick-pruned graph with non-aggressive pruned graph"""
```

Comment:
Is this test still valid now that pruning is always aggressive?

Reply:
Yes, the test is still valid (but the doc string is confusing). It is comparing `original_graph.prune` output with `quick_pruned_graph.prune` output. Just to check the quick prune graph does not change anything unexpectedly.